packages/contractkit/src/wrappers/Accounts.ts

```diff
@@ -237,8 +237,16 @@ export class AccountsWrapper extends BaseWrapper<Accounts> {
   /**
    * Returns the set name for the account
    * @param account Account
+   * @param blockNumber Height of result, defaults to tip.
    */
-  getName = proxyCall(this.contract.methods.getName)
+  async getName(account: Address, blockNumber?: number): Promise<string> {
+    return blockNumber
```

Comment:
you probably don't need the conditional here.. since you would be sending `undefined` when not set...
and code would be cleaner